src/test/java/com/teragrep/rlo_14/BenchmarkTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2010-2013, CloudBees Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.teragrep.rlo_14;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+import java.util.Calendar;
+import java.util.TimeZone;
+
+@EnabledIfSystemProperty(named="benchmark", matches="true")
+public class BenchmarkTest {
```

Comment:
jmh is better for benchmarking see https://www.baeldung.com/java-microbenchmark-harness